### ISSUE TITLE
SyntaxError in plugin 'gulp-babel'

### ISSUE THREAD

**Issue by daybugging** [S1SYPHOS/Gulp-Kirby-Starter-Kit#2]
```
Message:
    /your-project/main.js: 'import' and 'export' may only appear at the top level (60:1)
```

**Comment by S1SYPHOS** [comment]
I'll look into it!

**Comment by S1SYPHOS** [comment]
Fixed with ec4b7105cde0e4883d99f9d80c9df4d1bf3c5304.
Thanks for reporting!
